### ISSUE TITLE
feat: automate stackblitz

### ISSUE THREAD

**Issue by kyubisation** [sbb-design-systems/sbb-angular#400]
We want to provide a button to open examples on StackBlitz, similar to how Angular Material does it: https://material.angular.io/components/autocomplete/examples
We currently provide the source files of the examples under assets:
 - https://github.com/sbb-design-systems/sbb-angular/blob/master/src/showcase/assets/BUILD.bazel#L30
 - https://github.com/sbb-design-systems/sbb-angular/blob/master/src/showcase/app/shared/loader-builder.ts#L31 (example for highlighted sources)
See how Angular Material does it and try to extract, if possible. (Angular Material StackBlitz example: https://stackblitz.com/angular/yordnkjrxxb?file=src%2Fmain.ts)
This will probably require two similar StackBlitz templates for public and business.

https://stackblitz.com/docs

Also consider #362

*DoD*
- All examples have a "Open on StackBlitz" button

**Comment by vlanz** [comment]
I analysed this issue and found the following obstacles:

1. **Using the Stackblitz Javascript API**

> TL;DR: at the moment the Stackblitz JS API doesn't support the functionality we'd need to edit an existing project from a script.

The Stackblitz JS API seems to be only oriented towards embedding projects on a project or opening a new tab for a Stackblitz project. Since our interest is leaning more towards an "edit" functionality without a DOM present, these API endpoints aren't of much use. 
[DIrect link to the JS API docs](https://stackblitz.com/docs#javascript-sdk)

2. **Using the Stackblitz POST API**

> TL;DR: the POST API requires a token and a cookie in order to edit an existing project, which would be hard to acquire through a CI/CD script.

As @kyubisation tested, during a save operation a PATCH request is sent which requires a cookie and an authentication token to succeed. Acquiring said items proves to be a lengthy task in the context of a simple CI/CD script, therefore also rendering this option suboptimal.
[Direct link to the Stackblitz POST API](https://stackblitz.com/docs#post-api)

On a separate note, #362 can be fixed independently by creating a new example project implementing the business components.
